CMakeLists.txt

```diff
@@ -284,26 +274,44 @@ find_package(libuuid REQUIRED)
 
 find_package(libcurl REQUIRED)
 
-option(USE_CRYPTOPP "Cryptopp is ON" ON)
-find_package(cryptopp)
-if(CRYPTOPP_FOUND)
-        MESSAGE(STATUS "${CRYPTOPP_LIBRARIES}")
-        set(CRYPTO_LIBS ${CRYPTOPP_LIBRARIES})
-	set(USE_NSS 0)
-else()
-        MESSAGE(STATUS "Cryptopp not found using NSS instead")
-	find_package(NSS REQUIRED)
-        if(NSS_FOUND)
-		set(USE_NSS 1)
-		set(USE_CRYPTOPP 0)
-		find_package(NSPR)
-		if(NSPR_FOUND)
-			set(CRYPTO_LIBS ${NSS_LIBRARIES} ${NSPR_LIBRARIES})
-			#MESSAGE(STATUS "${CRYPTO_LIBS}")
-			#MESSAGE(STATUS "${NSS_INCLUDE_DIR} ${NSPR_INCLUDE_DIR}")
-		endif(NSPR_FOUND)
-	endif(NSS_FOUND)
-endif(CRYPTOPP_FOUND)
+option(WITH_NSS "Use NSS crypto and SSL implementations" OFF)
+if (${WITH_NSS})
+  find_package(NSS REQUIRED)
+  if(NSS_FOUND)
```

Comment:
if nss is not found, cmake will bail out with the default error message, so no need to check `NSS_FOUND` here?